slides/05_collaboration.qmd

```diff
@@ -69,8 +69,8 @@ Photo from <a href="https://unsplash.com/@timmossholder?utm_source=unsplash&utm_
 flowchart LR
   A[<font size=4> Kevin v1] --> B[<font size=4> Kevin v2]
   B --> C[<font size=4> Kevin v3]
-  B --> D[<font size=4> Daniel v1]
-  D --> E[<font size=4> Kevin + Daniel v4]
+  B --> D[<font size=4> Shuang v1]
+  D --> E[<font size=4> Kevin + Shuang v4]
   C --> E
```

Comment:
in line 95, suggest to use `SVN` to replace `SCM` (svn is version control tool, similar to git but scm is much bigger)

Reply:
+1

Reply:
Thank you, updated